src/Icons/StatusIcon.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+import Icon from './Icon';
+import { IStatusDot } from '..';
+
+const Container = styled.div`
+  position: relative;
+  display: inline-block;
+`;
+
+const StatusCounter = styled.div<{ color?: IStatusDot }>`
+  background-color: ${({ theme, color }) => color ? theme.colors.status[color] : 'hsla(0, 0%, 91.8%, 1.000)'};
```

Comment:
I think if there isn't a status color set for this we should make it transparent by default. It would be good to keep the fallback value but use that to set transparency. `'hsla(0, 0%, 91.8%, 0)'`

Then if we add an eased transition of around 750-1000ms perhaps, when this status icon changes the animation should help make that state change more noticeable but without being over the top.

If you feel it's better to do this with `opacity` for any reason then that works too. Maybe that helps with the numbers. Also if you think it's all a terrible idea, please feel free to let me know :)

Reply:
Hi!, I added the fallback and the transition.
Let me know how it looks.